### ISSUE TITLE
Filter by age

### ISSUE THREAD

**Issue by TCLynx** [RealRaven2000/quickFilters#247]
This is a feature request.  Please let me know if there is a way to do it that I haven't figured out yet.
I know I can create a filter that will move everything that is before or after a given date.  However, if I want to have a filter that automatically acts on messages meeting a particular criteria AND more than say 365 days old, I would have to edit and run that filter manually on a daily basis.

Thanks

**Comment by TCLynx** [comment]
Ok, I actually found it right after posting the above comment.
![image](https://github.com/RealRaven2000/quickFilters/assets/108405567/72ac9de8-b40d-449a-bd96-89a5b8d1de41)
Hum, now I just want to figure out how to make it automatically run on a particular folder without having to manually run it.

**Comment by RealRaven2000** [comment]
> Ok, I actually found it right after posting the above comment. !

> Hum, now I just want to figure out how to make it automatically run on a particular folder without having to manually run it.

Yeah so that's the problem - running automatically in any other folder than Inbox is difficult, I think it was possible with older versions of (now my) Add-on [FiltaQuilla](https://github.com/RealRaven2000/FiltaQuilla), but it's not currently supported - neither by Add-ons nor Thunderbird itself. You could add a [feature request there](https://github.com/RealRaven2000/FiltaQuilla/issues) to open the discussion.

**Comment by RealRaven2000** [comment]
> Ok, I actually found it right after posting the above comment. ![image](https://github.com/RealRaven2000/quickFilters/assets/108405567/72ac9de8-b40d-449a-bd96-89a5b8d1de41) Hum, now I just want to figure out how to make it automatically run on a particular folder without having to manually run it.

If you are using IMAP, check out #305
